### DISCUSSION TITLE
关于添加Mod加载器的问题

### DISCUSSION THREAD

**Discussion by wtommy932** [Meloong-Git/PCL#3553] [提问] [answered]
如果我已经下了一个没有任何Mod加载器的版本，我可以给它添加手动下载的Mod加载器吗？还是只能重新下一个附带Mod加载器的版本？

**Comment by allMagicNB** [comment, marked as answer]
只能重新下载……
更改的话已经在投票中了
#220（#227）

**Reply by wtommy932** [reply]
哦哦

**Reply by allMagicNB** [reply]
> 哦哦

能否给个 Mark as answer（

**Reply by wtommy932** [reply]
既然这样，那还要
![屏幕截图 2024-03-23 154827](https://github.com/Hex-Dragon/PCL2/assets/162002839/67d46708-33b6-4b76-bae3-e6b9c3501c2f)
干什么

**Reply by wtommy932** [reply]
不太懂这个是干什么的

**Reply by allMagicNB** [reply]
> 不太懂这个是干什么的

就是提供个手动安装的地方

**Reply by wtommy932** [reply]
> > 不太懂这个是干什么的
> 
> 就是提供个手动安装的地方

> 只能重新下载…… 更改的话已经在投票中了 #220（#227）

那手动安装又不能添加到MC里

**Reply by hejiehao** [reply]
这个可以把 Mod Loader 安装器下载下来
运行这个安装器可以安装 Mod Loader

**Reply by wtommy932** [reply]
> 这个可以把 Mod Loader 安装器下载下来 运行这个安装器可以安装 Mod Loader

好的

**Reply by wtommy932** [reply]
> > 这个可以把 Mod Loader 安装器下载下来 运行这个安装器可以安装 Mod Loader
> 
> 好的

让我试试
